src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -581,6 +771,10 @@ public static OptionData fromData(@Nonnull DataObject json)
         OptionType type = OptionType.fromKey(json.getInt("type"));
         OptionData option = new OptionData(type, name, description);
         option.setRequired(json.getBoolean("required"));
+        if (!json.isNull("min_value"))
+            option.setMinValue(json.getDouble("min_value"));
+        if (!json.isNull("max_value"))
+            option.setMaxValue(json.getDouble("max_value"));
```

Comment:
This will throw if type is INTEGER, probably need to check that

Reply:
added, though I think there's a prettier way than a ternary here.